docs/content/release_notes/changelog_1.8.0.md

```diff
@@ -0,0 +1,8 @@
+# 1.8.0 (December 2021)
+
+IBM® block storage CSI driver 1.8.0 adds new support and enhancements:
+- New support and enhancments for IBM Spectrum Virtualize family storage systems:
+    - New HyperSwap support
+    - New NVMe/FC support
+    - Increased StorageClass `SpaceEfficiency` parameter capabilities
+- Additional support for Kubernetes 1.23
```

Comment:
we are supporting K8S 1.21 and 1.22 fir 1.8.0

Reply:
done